ansible/files/ocp/mariadb/deployment.yaml

```diff
@@ -58,6 +58,15 @@ spec:
       initContainers:
       - name: mariadb-init
         image: docker.io/tripleomaster/centos-binary-mariadb:current-tripleo
+        command: ["/bin/bash"]
```

Comment:
This is fixed in the MariaDB-operator. Rather than an initContainer I used a k8s Job. https://github.com/openstack-k8s-operators/mariadb-operator/blob/master/pkg/dbinit.go

Reply:
lets merge this first and I will rebase